electron/app/index.ts

```diff
@@ -19,6 +19,8 @@ import {initTelemetry, saveInitialK8sSchema, setProjectsRootFolder} from './util
 
 const userHomeDir = app.getPath('home');
 const userDataDir = app.getPath('userData');
+app.commandLine.appendSwitch('ignore-certificate-errors');
```

Comment:
what does this do and what is it for?
can you add a comment to elaborate on these two lines so we know in the future when we read the code?